build-config/make/crosstool-ng.make

```diff
@@ -9,28 +10,38 @@
 # This is a makefile fragment that defines the build of crosstool-NG
 #
 
+# Default GCC version to build for the toolchain
+GCC_VERSION 		?= 4.9.2
+XTOOLS_LIBC 		?= uClibc
+XTOOLS_LIBC_VERSION 	?= 0.9.33.2
+
 CROSSTOOL_NG_DESC		= crosstool-NG
-CROSSTOOL_NG_VERSION		= 1.21.0
+CROSSTOOL_NG_LATEST_RELEASE_VERSION = 1.22.0
+CROSSTOOL_NG_VERSION		= 1.22.1
+CROSSTOOL_NG_COMMIT		= 11cb2ddd43fd1ff493f4b7cd63f1cf654294165f
 CROSSTOOL_NG_TARBALL		= crosstool-ng-$(CROSSTOOL_NG_VERSION).tar.xz
-CROSSTOOL_NG_URLS		+= $(ONIE_MIRROR) http://crosstool-ng.org/download/crosstool-ng
+CROSSTOOL_NG_URLS		+= $(ONIE_MIRROR) https://crosstool-ng.org/download/crosstool-ng
 CROSSTOOL_NG_BUILD_DIR		= $(BUILDDIR)/crosstool-ng
 CROSSTOOL_NG_STAMP_DIR		= $(CROSSTOOL_NG_BUILD_DIR)/stamp
 CROSSTOOL_NG_DIR		= $(CROSSTOOL_NG_BUILD_DIR)/crosstool-ng-$(CROSSTOOL_NG_VERSION)
 
 CROSSTOOL_NG_SRCPATCHDIR	= $(PATCHDIR)/crosstool-NG
-CROSSTOOL_NG_DOWNLOAD_STAMP	= $(DOWNLOADDIR)/crosstool-ng-download
+CROSSTOOL_NG_DOWNLOAD_STAMP	= $(DOWNLOADDIR)/crosstool-ng-$(CROSSTOOL_NG_VERSION)-download
 CROSSTOOL_NG_SOURCE_STAMP	= $(CROSSTOOL_NG_STAMP_DIR)/crosstool-ng-source
 CROSSTOOL_NG_PATCH_STAMP	= $(CROSSTOOL_NG_STAMP_DIR)/crosstool-ng-patch
 CROSSTOOL_NG_CONFIGURE_STAMP	= $(CROSSTOOL_NG_STAMP_DIR)/crosstool-ng-configure
 CROSSTOOL_NG_BUILD_STAMP	= $(CROSSTOOL_NG_STAMP_DIR)/crosstool-ng-build
 CROSSTOOL_NG_STAMP		= $(CROSSTOOL_NG_SOURCE_STAMP) \
 				  $(CROSSTOOL_NG_PATCH_STAMP) \
 				  $(CROSSTOOL_NG_CONFIGURE_STAMP) \
-				  $(CROSSTOOL_NG_BUILD_STAMP) 
+				  $(CROSSTOOL_NG_BUILD_STAMP)
```

Comment:
There will always be cosmetic things in the existing code (like this whitespace) that we can fix up.

However, if you want to clean up cosmetic things, then make that a separate "cosmetic only" commit.

Mixing cosmetic commits with functional commits is a bad practice and creates a muddled git history.